### ISSUE TITLE
Photo resolution sufficient for printing?

### ISSUE THREAD

**Issue by tbar0970** [tbar0970/jethro-pmm#505]
Assuming the GD PHP extension is installed, Jethro resizes photos when they are uploaded, I think to max width of 200 and/or max height of 200.  This is good for saving space.

But we also have a feature where you can include photos in the DOCX version of a contact list which will get printed.  I suspect 200x200px images will not be good enough quality for printing.  Might need to improve the resolution we store the photos at.

This would then raise the question if we should store both a "small" and "medium" version of the photo so that the version used in the browser can still be small and quick to load (esp with lots on a page).

**Comment by GatesheadBen** [comment]
I agree. And storing a 200x200px thumbnail and a printable version sounds like a good idea too.
